src/sources/dataModel/taxonomic-service.json

```diff
@@ -210,72 +443,194 @@
+                            "https://orcid.org/0000-0001-9790-9277"
                         ]
                     },
+                    "schema:affiliation": {
+                        "type": "object",
+                        "properties": {
+                            "@type": {
+                                "type": "string",
+                                "description": "The type of the affiliation",
+                                "const": "schema:Organization"
+                            },
+                            "schema:identifier": {
+                                "type": "string",
+                                "description": "A unique identifier to identify the author's affiliation; ROR identifiers are valid",
+                                "examples": [
+                                    "https://ror.org/0566bfb96"
+                                ]
+                            },
+                            "schema:name": {
+                                "type": "string",
+                                "description": "Full name of the author's affiliated orgnisation",
+                                "examples": [
+                                    "Naturalis Biodiversity Center"
+                                ]
+                            }
+                        },
+                        "required": [
+                            "@type",
+                            "schema:identifier"
+                        ],
+                        "additionalProperties": false
+                    }
+                },
+                "required": [
+                    "@type",
+                    "schema:identifier",
+                    "schema:affiliation"
+                ],
+                "additionalProperties": false
+            }
+        },
+        "schema:Maintainer": {
+            "type": "array",
+            "description": "An array of maintainers of a Dataset, software package (SoftwareApplication), or other Project. A maintainer is a Person or Organization that manages contributions to, and/or publication of, some (typically complex) artefact.",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "@type": {
+                        "type": "string",
+                        "description": "The type of the maintainer",
+                        "const": "schema:Person"
+                    },
+                    "schema:identifier": [
+                        {
+                            "type": "string",
+                            "description": "A unique identifier to identify the maintainer; GitHub identifiers are valid",
+                            "examples": [
+                                "https://api.github.com/users/username"
+                            ]
+                        },
+                        {
+                            "type": "string",
```

Comment:
Something is going wrong here, this is invalid JSON Schema

Reply:
Don't know if its completely valid, but changed it to:
```
"schema:identifier": {
                        "type": "array",
                        "items": {
                            "type": "string",
                            "description": "Unique identifiers to identify the maintainer; GitHub and ORCID identifiers are accepted",
                            "examples": [
                                "https://api.github.com/users/username",
                                "https://orcid.org/0000-0001-9790-9277"
                            ]
                        }
                    },
```